test/actions/profile.spec.js

```diff
@@ -37,7 +37,7 @@ describe('updateComment', () => {
     expect(typeof updateComment).toBe('function');
   });
 
-  it('should create an appropiate action', () => {
+  it('should create an appropriate action', () => {
```

Comment:
lets get simple in our wording 

```suggestion
  it('should create an action', () => {
```